microsoft-edge/dev-videos/index.md

```diff
@@ -625,16 +625,12 @@ December 9, 2021
 
 [![Thumbnail image for video "Debug memory leaks with the Microsoft Edge Detached Elements tool"](./index-images/detached-elements.png)](https://www.youtube.com/watch?v=v2iy17ptmBk)
 
-To debug DOM memory leaks, use either:
-* The **Detached Elements** tool.  Detached nodes have a link to the associated JavaScript code.
-* The **Detached elements** profiling type in the **Memory** tool.
-
-The **Detached Elements** tool helps you investigate and resolve DOM memory leaks.
+March 7, 2025: The **Detached Elements** tool is about to be removed.  To debug DOM memory leaks, use the **Detached elements** profiling type in the **Memory** tool.
```

Comment:
This PR removes the **Detached Elements** tool documentation, so I think _"... is about to be removed ..."_ is a little weak. I would either change to _"... has been removed ..."_, or omit the paragraph altogether.